### ISSUE TITLE
Left hand mode

### ISSUE THREAD

**Issue by Aveeux** [8bitPit/Niagara-Issues#12]
![left](https://user-images.githubusercontent.com/11475502/44894332-d0003280-ad21-11e8-9da8-73cb011880f4.jpg)

It would be great if there would be a toggle to switch the A-Z scroll to the left since it's pretty hard for left handeds to use it as some screens are too big. I have attached a mock up. Thanks!

**Comment by timm0e** [comment]
Hey there,

You can Already scroll on the left, the indicator will appear when you start scrolling :)

**Comment by timm0e** [comment]
But maybe this isn't clear enough at first, which is kind of a design flaw, so I'll mark this as a feature request :+1:

**Comment by Maxr1998** [comment]
@jakcharvat a toggle to flip the visible scroll-bar would be very nice, true. Kinda interesting this doesn't exist yet considering both the dev and myself are left-handed :thinking:

**Comment by 8bitPit** [comment]
The circled Letter would overlap with the app icons, so that’s why I made it scroll, but not be visible on the left side.

**Comment by 587saboor** [comment]
@8bitPit maybe you could make everything flipped altogether? With the clock, apps, etc on the right and the scrollbar on the left

so basically whatever is on the right goes on the left and whatever is on the left goes on the right

**Comment by 8bitPit** [comment]
I tried that, and it looks odd.

**Comment by jakcharvat** [comment]
In that case, maybe the left-handed scrollbars could be simpler, maybe just a list of letters down the side that doesn't come out on scroll like the one on the right does, and the current letter is highlighted ON the scrollbar. I'll try and make an illustration so you get what I mean

**Comment by jakcharvat** [comment]
https://photos.app.goo.gl/1kTK9H7QvvpLcQ4W7 @8bitPit

Made on my phone as I don't have my laptop on me at this moment, don't judge 😂

**Comment by 8bitPit** [comment]
Thank you, maybe I show two scrollbars when you touch the left side. I'll send you a picture when I am working on this.

**Comment by jakcharvat** [comment]
Awesome, thanks

**Comment by zamber** [comment]
> I tried that, and it looks odd.

I'm left-handed but I use my phone mosty with my right hand. Go figure :).

Still, I'm for this mirroring feature. It will be needed either ways whenever you would want to implement support for RTL languages (that also why it looks odd, we're used to LTR).

**Comment by 8bitPit** [comment]
@zamber yes, it will be added for RTL languages, but I did not plan to add a toggle in Niagara's settings (so the layout is only RTL based on your language / country)

**Comment by Seralyn** [comment]
Just wanted to chime in on this and say that I, too, hope for a mirrored version. As a left hander, I came to this forum to request this feature and found out that I can use the left side, which is cool, but my finger covers up and blocks the icons and names of the apps. So, I would very much like to have that feature! I get that it probably looks weird to you since you've been seeing it the way it currently is for so long but for someone just coming into it, I haven't had time to become accustomed to seeing it one way and when I mirrored it in photoshop it seemed equally fine to me. 

Other than that, just want to say keep up the great work and I'm so excited to see where this goes! <3

**Comment by cauerego** [comment]
as a kind of both hands person, i see zero need for this, because the left side already works fine.

however, it only works because i know it does. and sometimes i even forget about it! some kind of visual clue would be very welcomed. it could be something really simple like a line or small like a star on the top and circle in the bottom, to mirror the alphabet end points.

**Comment by 8bitPit** [comment]
@cauerego I like your idea. What about a little hint or tutorial message that shows up if the user hasn't yet scrolled on the left side?

**Comment by cauerego** [comment]
@8bitpit tutorial might help a lot of people, but i'd still keep forgetting about it until i get completely used with it.

since i don't use just my left hand and i hardly use the sidescroll to navigate, it's probably much harder for me to get used. plus i did stop using Niagara for a while. so, yeah, almost an year after i know about it, i still get myself using the right scroll with my left hand, because that's where the visual cue directs me to.

any kind of constant simple reminder that the left side also works just like the right side... perhaps even something new that would be repeated on the right side. sorry I'm not much of a visual person or mocking up sketches. 😁

**Comment by mandorlinoGitHub** [comment]
Please add a left-handed mode. This would be a game-changer!

**Comment by 8bitPit** [comment]
Hey everyone 👋 We concluded not to add a dedicated left-hand-mode, at least not soon. I'm sorry if you were waiting for this. I'm not a fan of mirroring the complete layout because most people are just so used to everything being left-aligned or centered. In my opinion, a right-aligned list looks uncomfortable, and I haven't seen any app yet that right aligns their app list. Moreover, since v0.34, Every new user now sees a tip explaining they can also scroll through their app list on the left side. Since we added the tip, we got fewer requests for a left-hand-mode.
